docs/integrations/sources/exchangeratesapi.md

```diff
@@ -46,6 +46,7 @@ If you have `free` subscription plan \(you may check it [here](https://manage.ex
 
 | Version | Date | Pull Request | Subject |
 |:--------| :--- | :--- | :--- |
+| 0.2.7   | 2022-08-23 | [15884](https://github.com/airbytehq/airbyte/pull/15884) | Added support for new API Layer endpoint |
```

Comment:
@sajarin the dockerfile version is not matching with the version in the document.

Reply:
Should be good to go now?